synth.metadata

```diff
@@ -12,15 +12,15 @@
       "git": {
         "name": "googleapis",
         "remote": "https://github.com/googleapis/googleapis.git",
-        "sha": "07883be5bf3c3233095e99d8e92b8094f5d7084a",
-        "internalRef": "247530843"
+        "sha": "99efb1441b7c2aeb75c69f8baf9b61d4221bb744",
+        "internalRef": "248724297"
       }
     },
     {
       "template": {
         "name": "node_library",
         "origin": "synthtool.gcp",
-        "version": "2019.5.2"
+        "version": "2019.4.10"
```

Comment:
Is this version change something to be concerned about?

Reply:
Nope!  This is just flagging the version of `synthtool` used to generate the change.  This thing is mostly just a machine readable log file 🤷‍♂